### ISSUE TITLE
Documentation about postgres JDBC DATABASE_URL needs improvement

### ISSUE THREAD

**Issue by nerdinand** [hanami/model#452]
When generating a new hanami app with JRuby and `--database postgres`, the resulting `DATABASE_URL` variable in `.env.development` looks something like this:

```
DATABASE_URL="jdbc:postgresql://localhost/bookshelf_development"
```

The documentation on http://hanamirb.org/guides/1.0/models/database-configuration/ suggests adding user/password like this:

```
DATABASE_URL="postgresql://username:password@localhost/bookshelf_development"
```

however, this will not work in the case of jdbc. After a bunch of googling, I figured out user and password need to be added this way: 

```
DATABASE_URL="jdbc:postgresql://localhost/bookshelf_development?user=my_user&password=my_password"
```

The documentation should include this hint to avoid other people encountering this pitfall.

**Comment by AlfonsoUceda** [comment]
thanks @nerdinand we need to test this because not long ago I improved url postgresql connection https://github.com/hanami/model/pull/441 allowing params for the configuration

**Comment by AlfonsoUceda** [comment]
@nerdinand I've tested in develop branch so this is resolved in 1.1.0.beta3. So in 1.1.0 this fix will be available. I'm going to update the docs for the new version
